### ISSUE TITLE
[Accessibility, a11y] Quick picks select many - needs a descriptive select all checkbox screen reader callout

### ISSUE THREAD

**Issue by MicroFish91** [microsoft/vscode#149884]
**VS Code Version**: 1.67.1

**Operating System**: Windows 11

**Screen Reader**: NVDA

**Extensions Installed**:
Put together an extension to quickly demo the bare VS Code UI features without our normal SDK and extension wrappers.
"a11y-kit" (https://marketplace.visualstudio.com/items?itemName=MicroFish.accessibility-testkit)

**Repro Steps**:

1. Install a11y-kit from VS Code Marketplace
2. Open command palette and run a11y-kit's menu command
3. Next, select: "Show Quick Picks - Many" and observe.

**Expected**:
There should be a way for the screen reader to inform the user that the checkbox to the left of the main input field is used to select all quick pick items.  I suppose the screen reader callout might read to the user something like... "select all items checkbox, unchecked"

**Actual**:
When entering the quick pick selection screen, all the quick pick items are listed as check-able items as you select over them; however, the select all checkbox next to the main input box has no such similar screen reader callout.  When highlighting directly using keyboard, it states that it is a checkbox but does not describe its specific function in the context of the section.

Attachments:
![sqp_many](https://user-images.githubusercontent.com/40250218/169182010-894075ed-e288-4bfe-b67d-4ff266d8cc62.gif)

Thank you!
